Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -243,16 +242,32 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
   using value_type = typename decltype(spContainer)::SpacePointProxyType;
   using seed_type = Acts::Seed<value_type>;
 
-  // extent used to store r range for middle spacepoint
-  Acts::Extent rRangeSPExtent;
-
   Acts::CylindricalSpacePointGrid<value_type> grid =
       Acts::CylindricalSpacePointGridCreator::createGrid<value_type>(
           m_cfg.gridConfig, m_cfg.gridOptions);
 
   Acts::CylindricalSpacePointGridCreator::fillGrid(
       m_cfg.seedFinderConfig, m_cfg.seedFinderOptions, grid,
-      spContainer.begin(), spContainer.end(), rRangeSPExtent);
+      spContainer.begin(), spContainer.end());
+
+  // Compute radius Range
+  // we rely on the fact the grid is storing the proxies
+  // with a sorting in the radius
+  double minRange = std::numeric_limits<double>::max();
+  double maxRange = std::numeric_limits<double>::lowest();
+  for (const auto& coll : grid) {
+    if (coll.empty()) {
+      continue;
+    }
+    const auto* firstEl = coll.front();
+    const auto* lastEl = coll.back();
+    if (firstEl->radius() < minRange) {
+      minRange = firstEl->radius();
+    }
+    if (lastEl->radius() > maxRange) {
+      maxRange = lastEl->radius();
+    }
```

Comment:
Nitpick, this could be
```suggestion
    minRange = std::min(firstEl->radius(), minRange);
    maxRange = std::max(lastEl->radius(), maxRange);
```

I believe.

Reply:
good point. Done